Core/Config.h

```diff
@@ -394,6 +394,8 @@ struct Config {
 	bool bEnableAdhocServer;
 	int iWlanAdhocChannel;
 	bool bWlanPowerSave;
+	bool bEnableNetworkChat;
+	int iNewChat;
```

Comment:
iNewChat is a counter used when chatscreen is not visible to notify user there is a chat coming from other user. if the chatscreen is visible it will not increment it and will call the pointer to update the chatsreen.  the counter is displayed on the  choice with value display https://github.com/hrydgard/ppsspp/pull/9090/files#diff-4497b3f963a2b3d65020d48fb909f4adR771 . im just copying another choice with value display line look at the code, and found its used config.h so i use that file for global too. ah i didn't notice that line inside and part of ppsspp config struct. should i move it to proadhoc.h and extren that variable so i can used that new chat variable on emuscreen and chatscreen? move it to proAdhoc.h it is the right choice ?